File: app/src/main/java/com/example/login/MODELS/Pagination.java
package com.example.login.MODELS;

import com.google.gson.annotations.SerializedName;
import java.io.Serializable;

public class Pagination implements Serializable {
    @SerializedName("totalCount")
    private int totalCount;
    @SerializedName("totalPages")
    private int totalPages;
    @SerializedName("currentPage")
    private int currentPage;

    public int getTotalCount() {
        return totalCount;
    }

    public int getTotalPages() {
        return totalPages;
    }

    public int getCurrentPage() {
        return currentPage;
    }
}

File: provider-app/app/src/main/java/com/example/providerapp/data/model/Pagination.kt
package com.example.providerapp.data.model

import com.google.gson.annotations.SerializedName

data class Pagination(
    @SerializedName("totalCount")
    val totalCount: Int,

    @SerializedName("totalPages")
    val totalPages: Int,

    @SerializedName("currentPage")
    val currentPage: Int
)